### PR TITLE
[Enhancement] avoid mockup be

### LINKED ISSUE #17537: [Enhancement] unstable FE ut case
### Steps to reproduce the behavior (Required)
Unstable FE ut case : test/java/com/starrocks/alter/RollupJobV2Test.java : testSchemaChange1

```
Missing 1 invocation to:
com.starrocks.system.Backend#isAlive()
   on mock instance: com.starrocks.system.Backend@78f2029d
Caused by: Missing invocations
	at com.starrocks.system.Backend.isAlive(Backend.java)
	at com.starrocks.task.AgentBatchTask.run(AgentBatchTask.java:161)
	at java.lang.Thread.run(Thread.java:750)
```

### StarRocks version (Required)
 - main

### PR DESCRIPTION
**Pull request by luohaha**
Signed-off-by: luohaha <18810541851@163.com>

## What type of PR is this：
- [] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #17537

## Problem Summary(Required) ：

Fix unstable FE ut : test/java/com/starrocks/alter/RollupJobV2Test.java : testSchemaChange1
Avoid mockup be, or it will expected `isAlive` will be called, but in `RollupJobV2Test.java` isn't be called, so it faill with:
```
17:28:50 [ERROR] Tests run: 5, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 47.445 s <<< FAILURE! - in com.starrocks.alter.RollupJobV2Test
17:28:50 [ERROR] testSchemaChange1(com.starrocks.alter.RollupJobV2Test)  Time elapsed: 7.121 s  <<< ERROR!
17:28:50 Missing 1 invocation to:
17:28:50 com.starrocks.system.Backend#isAlive()
17:28:50    on mock instance: com.starrocks.system.Backend@78f2029d
17:28:50 Caused by: Missing invocations
17:28:50         at com.starrocks.system.Backend.isAlive(Backend.java)
17:28:50         at com.starrocks.task.AgentBatchTask.run(AgentBatchTask.java:161)
17:28:50         at java.lang.Thread.run(Thread.java:750)
```

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: fe/fe-core/src/test/java/com/starrocks/load/ExportCheckerTest.java [hunk 1]
```diff
@@ -49,13 +49,6 @@ public Backend getBackend(long backendId) {
             }
         };
 
-        new MockUp<Backend>() {
```
Comment (nshangyiming) [inline fe/fe-core/src/test/java/com/starrocks/load/ExportCheckerTest.java:52]: does this mockup only affect current case or any other case?

**Review by Astralidea — APPROVED** [review]

File: fe/fe-core/src/test/java/com/starrocks/load/ExportCheckerTest.java (hunk 1 above)
Comment (luohaha) [reply fe/fe-core/src/test/java/com/starrocks/load/ExportCheckerTest.java:52]: It seems like it affect other case, check this: 
```
17:28:50 [ERROR] testSchemaChange1(com.starrocks.alter.RollupJobV2Test)  Time elapsed: 7.121 s  <<< ERROR!
17:28:50 Missing 1 invocation to:
17:28:50 com.starrocks.system.Backend#isAlive()
17:28:50    on mock instance: com.starrocks.system.Backend@78f2029d
17:28:50 Caused by: Missing invocations
17:28:50         at com.starrocks.system.Backend.isAlive(Backend.java)
17:28:50         at com.starrocks.task.AgentBatchTask.run(AgentBatchTask.java:161)
17:28:50         at java.lang.Thread.run(Thread.java:750)
```

**Review by nshangyiming — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by gengjun-git — APPROVED** [review]

**Pull request merged**